frontend/src/modules/history/components/Translation.js

```diff
@@ -251,172 +238,204 @@ export class TranslationBase extends React.Component<InternalProps, State> {
             className += ' can-reject';
         }
 
-        let canDelete = (canReview || ownTranslation) && !isReadOnlyEditor;
-        let canReject = (canReview || (ownTranslation && !translation.approved)) && !isReadOnlyEditor;
+        let canDelete = (isTranslator || ownTranslation) && !isReadOnlyEditor;
+        let canReject = (isTranslator || (ownTranslation && !translation.approved)) && !isReadOnlyEditor;
+        let canComment = (isTranslator || ownTranslation);
```

Comment:
According to the "Permissions" section of the [spec](https://docs.google.com/document/d/1W_MEYwtNn-_J0Uiu7gTSwc2B3oux75rjVtjGu-YoFi4/edit#heading=h.1frxkg572534), any logged in contributor `canComment`, so the value should be `user.isAuthenticated`. And then you don't need the `!!` part of the `!!canComment` below.

I think it's still valuable to have the `canComment` variable defined along with other `CanSomething` permissions.